src/components/meeting/MeetingFab.js

```diff
@@ -23,6 +24,12 @@ export const MeetingFab = (props) => {
             >
                 <DashboardIcon />
             </Action>
+            <Action
+                text="Task"
```

Comment:
Może głupie pytanie o co chodzi z text=Task?

Reply:
Yyyy ogólnie jest tak, że ten action albo potrzebuje, albo dostaje propa task i gdzieś go wyświetla. Ogólnie chodzi o to, żeby było gdzieś wskazanie, że jest to task, jakby komuś ikonka nie wystarczyła XDD